src/lib/twitter/notifyTwitter.ts

```diff
@@ -0,0 +1,31 @@
+import TwitterAPI from 'twitter-api-v2';
+// import { fileTypeFromBuffer } from "file-type";
+import { NFTSale, SaleMethod } from "lib/marketplaces";
+import axios from 'axios';
+
+export default async function notifyTwitter(twitterClient: TwitterAPI, nftSale: NFTSale) {
+    const nftName = nftSale.nftData?.name;
+    const text = `${nftName} has just been sold ${
+        nftSale.method === SaleMethod.Bid ? "via bidding " : ""
+    }for ${nftSale.getPriceInSOL()} S◎L at ${nftSale.marketplace.name}! #SolanaNFTs #NFTSale`
+    const mediaArr: string[] = [];
+    if (Boolean(nftSale.nftData?.image)) {
+        const data = await getImageDataFromUrl(nftSale.nftData?.image as string);
+        const media = await twitterClient.v1.uploadMedia(data, { type: await getDataType(data) });
+        mediaArr.push(media);
+    }
+    return twitterClient.v1.tweet(text, {
+        media_ids: mediaArr
+    })
+}
+
+async function getDataType(buffer: Buffer) {
+    const { fromBuffer } = await import("file-type");
```

Comment:
How come you need to import it here?

Reply:
Would love to just do `import { fromBuffer } from "file-type"` at the top but due to the below issue, we can't.

https://github.com/sindresorhus/file-type/issues/525

Reply:
😢 fair enough. 
I think leave a comment above the line with the reference to the issue would be nice for the next person who read it.